### PR TITLE
hyperfine: add page

### LINKED ISSUE #2330: page request: hyperfine
Add documentation for [hyperfine](https://github.com/sharkdp/hyperfine)

### PR DESCRIPTION
**Pull request by russsaidwords**
Fixes #2330.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/hyperfine.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# hyperfine
+
+> A command-line benchmarking tool.
+
+- Run a basic benchmark:
+
+`hyperine '{{make}}'`
```
Comment (jedahan) [inline pages/common/hyperfine.md]: missing f?

Comment (russsaidwords) [reply pages/common/hyperfine.md]: Indeed, sorry about that, editing now...

File: pages/common/hyperfine.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# hyperfine
+
+> A command-line benchmarking tool.
+
+- Run a basic benchmark:
+
+`hyperfine '{{make}}'`
+
+- Change minimum number of benchmarking runs:
```
Comment (agnivade) [inline pages/common/hyperfine.md]: What is the default number

Comment (pxgamer) [reply pages/common/hyperfine.md]: In the docs it says
>By default, it will perform _at least_ 10 benchmarking runs.

Which is... a bit confusing.

Comment (agnivade) [reply pages/common/hyperfine.md]: Got it. It keeps on running the benchmarks until it has achieved a minimum variance to decide that the benchmark is stable. I just wanted the no. 10. We should clarify that.

Comment (pxgamer) [reply pages/common/hyperfine.md]: Oh ok 👍

**Pull request merged**